kratos/geometries/hexahedra_3d_8.h

```diff
@@ -580,6 +580,27 @@ template<class TPointType> class Hexahedra3D8 : public Geometry<TPointType>
         return edges;
     }
 
+    /** This method calculates and returns the average edge length of the geometry
+    *
+    * @return double value with the average edge length
+    *
+    */
+    double AverageEdgeLength() const override 
+    {
+        return (MathUtils<double>::Norm3(this->GetPoint(0)-this->GetPoint(1)) +
```

Comment:
Can you please get the coordinates first and then compute, I think it will be more efficient

Reply:
done!